open_gpt/cli/commands/serve.py

```diff
@@ -56,6 +59,7 @@ def handle(self) -> int:
             cors=self.option('enable_cors'),
             uses_with={
                 'precision': self.option('precision'),
+                'adapter_name_or_path': self.option('adapter_name_or_path'),
```

Comment:
The executor does not recognize this param yet. Please also update it in this PR.